tests/setup_test.go

```diff
@@ -3664,3 +3671,168 @@ func TestExportImport(t *testing.T) {
 		t.Fatalf("expected 0 orgs got: %d", len(orgs))
 	}
 }
+
+func TestRunEventsNotification(t *testing.T) {
+	tests := []struct {
+		name                   string
+		config                 string
+		expectedRunResult      rstypes.RunResult
+		expectedRunPhase       rstypes.RunPhase
+		expectedRunPhaseEvents []rstypes.RunPhase
```

Comment:
you should also check the run id it if matches and perhaps the various tasks statuses.